networkx/algorithms/link_analysis/pagerank_alg.py

```diff
@@ -231,6 +265,8 @@ def google_matrix(
         p = np.repeat(1.0 / N, N)
     else:
         p = np.array([personalization.get(n, 0) for n in nodelist], dtype=float)
+        if p.sum() == 0:
```

Comment:
Won't trying to divide by 0, raise a ZeroDivisionError?

Reply:
Surprisingly no! A numpy array will be created with nan.
```
>>> arr = np.array([0., 0., 0., 0.])
>>> arr /= arr.sum()
>>> arr
array([nan, nan, nan, nan])
```

Reply:
Thanks.